### ISSUE TITLE
InAppWebView alert() not work

### ISSUE THREAD

**Issue by xzr9527** [pichillilorenzo/flutter_inappwebview#133]
HTML code:
```
<p>
        <input type="button" id="enter" value="call" onclick="testClick();" />
</p>

 js code:
    function testClick() {
             alert("hello");
    }

``` 
the code in Chrome work normally. but in InAppWebView do not work. do you know how to resolve the problem . Thanks

**Comment by xzr9527** [comment]
@pichillilorenzo

**Comment by pichillilorenzo** [comment]
Now it should be fixed in `2.0.0`. Also, this plugin changed its name to `flutter_inappwebview`. The current latest version now is `2.1.0+1`. So, you can change your dependency in`flutter_inappwebview: ^2.1.0+1`.
